src/utils/dataDisposal.utils.ts

```diff
@@ -0,0 +1,51 @@
+import { APIRequestContext } from '@playwright/test';
+import { CustomersApiService } from 'api/services/customers.api-service';
+import { ProductsApiService } from 'api/services/product.api-service';
+import { OrdersAPIService } from 'api/services/orders.api-service';
+
+export class DataDisposalUtils {
+  private ordersApiService: OrdersAPIService;
+  private customersApiService: CustomersApiService;
+  private productsApiService: ProductsApiService;
+
+  constructor(context: APIRequestContext) {
+    this.ordersApiService = new OrdersAPIService(context);
+    this.productsApiService = new ProductsApiService(context);
+    this.customersApiService = new CustomersApiService(context);
+  }
+
+  async clearOrders(orderId: string, token: string) {
```

Comment:
Возможно сделать массив в параметре, так как возможны кейсы с несколькими ордерами или различными кастомерами
```suggestion
  async clearOrders(orderId: string[], token: string) {
```